ui/__mocks__/actions.js

```diff
@@ -21,10 +30,33 @@ module.exports = {
   ...actions,
   getTokenStandardAndDetails: (address) => {
     if (!TOKEN_DETAILS_MOCK[address]) {
-      return Promise.reject(new Error(`Token not found: ${address}`));
+      console.log('Token not found:', address);
```

Comment:
Was this `console.log` left intentionally, or is it meant for debugging and should be removed?